docs/introduction.md

```diff
@@ -250,8 +411,9 @@ You can then copy part of this data folder to be used in DPS:
 
 You can then run the DPS, and it should properly build its index based on the given information.
 
-### More Resources
+## More Resources
 
 * [Flow Technical Papers](https://www.onflow.org/technical-paper)
 * [Flow Developer Documentation](https://docs.onflow.org/)
 * [Flow Developer Discord Server](https://onflow.org/discord)
+* [Scalability Trilemma](https://aakash-111.medium.com/the-scalability-trilemma-in-blockchain-75fb57f646df)
```

Comment:
```suggestion
* [Scalability Trilemma](https://en.wikipedia.org/wiki/Zooko%27s_triangle)
```

Reply:
I checked this again, and I think they don't really mean the same thing.

Zooko's triangle: Human-meaningful, decentralized, secure
Scalability trilemma: Scalability, decentralization, security

I didn't see a wiki for the `scalability trilemma` (or `blockchain trilemma` as I've seen it too). Btw this link was already in the doc, it was just moved to the bottom after the content was changed and I didn't think it really fit in the previous place.

Reply:
```suggestion
* [Scalability Trilemma](https://vitalik.ca/general/2021/04/07/sharding.html#the-scalability-trilemma)
```